src/main/java/com/carleton/CapstoneSystem/Time.java

```diff
@@ -0,0 +1,90 @@
+package com.carleton.CapstoneSystem;
+
+
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+public class Time extends Date {

```

Comment:
is there a reason why we are having a time class. Java already has a Date class. We don't need to know about the hours and min?

Also, why does time have a description?

Reply:
the description is to show the date like :"yyyy/mm/day hh:mm", I tried to have the date class but the data base did not work